src/main/java/fr/skyost/owngarden/worldedit/WorldEditUtils.java

```diff
@@ -0,0 +1,196 @@
+    }
+
+    /**
+     * Returns the file associated with the given schematic.
+     *
+     * @param schematic The schematic.
+     *
+     * @return The file.
+     */
+    private File getFile(final String schematic) {
+        return new File(plugin.schematicsDirectory, schematic);
+    }
+
+    /**
+     * Loads a schematic.
+     *
+     * @param path The schematic file (must be in the schematics directory).
+     *
+     * @return The WorldEdit clipboard holder.
+     *
+     * @throws IOException If any I/O exception occurs.
+     */
+
+    private ClipboardHolder loadSchematic(final Path path) throws IOException {
+        if (!Files.exists(path)) {
+            throw new IOException("Schematic not found : " + path.toString());
+        }
+        final ClipboardFormat format = ClipboardFormats.findByFile(new File(path.toUri()));
+        if (format == null) throw new IOException("Unknown schematic format : " + path.toString());
+        return new ClipboardHolder(format.getReader(new BufferedInputStream(Files.newInputStream(path))).read());
+    }
+
+    /**
+     * Grows a tree at the specified location.
+     *
+     * @param schematic The schematic file.
+     * @param location The location.
+     *
+     * @return Whether the operation has been a success.
+     */
+
+    public boolean growTree(final Path schematic, final Location location) {
+        try {
+            location.getBlock().setType(Material.AIR, false);
+            final ClipboardHolder holder = loadSchematic(schematic);
+            if (plugin.randomRotation) {
+                final int degrees = plugin.rnd.nextInt(4) * 90;
+                if (degrees != 0) {
+                    final AffineTransform transform = new AffineTransform();
+                    transform.rotateY(degrees);
+                    holder.setTransform(transform);
+                }
+            }
+            final Clipboard clip = holder.getClipboards().get(0);
+            final BlockVector3 dimensions = clip.getDimensions();
+            if (plugin.checkHeight && !checkHeight(dimensions, location)) {
+                return false;
+            }
+            clip.setOrigin(BlockVector3.at(dimensions.getBlockX() >> 1, 0, dimensions.getBlockZ() >> 1));
+            final EditSession session = WorldEdit.getInstance().newEditSession(new BukkitWorld(location.getWorld()));
+            final Operation operation = holder
+                .createPaste(session)
+                .to(BlockVector3.at(location.getBlockX(),
+                    location.getBlockY(), location.getBlockZ()))
+                .ignoreAirBlocks(true)
+                .build();
+            Operations.completeLegacy(operation);
+            session.close();
+
+            return true;
+        } catch (IOException | MaxChangedBlocksException ex) {
+            plugin.logger.info(Component.text("Unable to load the schematic : " + schematic.toString(), NamedTextColor.RED));
+            ex.printStackTrace();
+        }
```

Comment:
Remove ex.printStackTrace and handle the exception fully. 
To handle right, ask your self with: Can the run the plugin still if it fails here ?